src/main/java/com/meetup/server/startpoint/implement/StartPointProcessor.java

```diff
@@ -85,4 +89,8 @@ public void delete(StartPoint startPoint) {
     public void deleteAllByEvent(Event event) {
         startPointRepository.deleteAllByEvent(event);
     }
+
+    public void delete(User user) {
```

Comment:
P3) User 제거 시 StartPoint가 전부 제거되는 거니까 메소드명을 좀 더 구체화해도 좋을 거 같아요

```suggestion
    public void deleteAllByUser(User user) {
```